spec/workspace-spec.js

```diff
@@ -887,6 +887,30 @@ describe('Workspace', () => {
           ).toEqual([2, 11])
         )
       })
+
+      it('unfolds the fold containing the line', () => {
+        let editor
+
+        waitsForPromise(() =>
```

Comment:
A small improvement: you can make the spec function `async` to be able to use `await` on the `workspace.open()` function calls and then get rid of all the `runs()` and `waitsForPromise()` calls.

Reply:
@rafeca: Thanks for that suggestion. 4ac6107f953f4d83ea003f7dd2ce7be1ac08dab7 updates the test to adopt this approach. 

So. Much. Better. 😻